### ISSUE TITLE
pinning column under native html table

### ISSUE THREAD

**Issue by elaijuh** [josebalius/react-grid#1]
@josebalius , have you already got some thoughts on this?

**Comment by josebalius** [comment]
@elaijuh yes probably going to be something like this: https://www.datatables.net/extensions/fixedcolumns/
